### ISSUE TITLE
Error: Cannot find module 'react/lib/ReactComponentTreeHook' from 'ReactDebugTool.js'

### ISSUE THREAD

**Issue by rei-gun** [lukechilds/react-jsdom#7]
```   "enzyme": "^3.3.0",
    "enzyme-adapter-react-16": "^1.1.1",
    "enzyme-to-json": "^3.3.1",
    "eslint": "^3.12.2",
    "eslint-plugin-babel": "^4.1.2",
    "eslint-plugin-flowtype": "^2.46.1",
    "eslint-plugin-jsx-a11y": "^3.0.2",
    "eslint-plugin-react": "^6.8.0",
    "jest": "^22.4.3",
    "react-addons-test-utils": "^15.6.2",
    "react-scripts": "0.6.1",
    "react-test-renderer": "^16.2.0",
    "redux-mock-store": "^1.5.1",
  },
  "dependencies": {
    "react": "^16.2.0",
    "react-addons-css-transition-group": "^15.3.2",
    "react-addons-transition-group": "^15.6.0",
    "react-countdown-now": "^1.3.0",
    "react-datetime": "^2.14.0",
    "react-dom": "^16.2.0",
    "react-redux": "^5.0.7",
    "react-redux-loading-bar": "^2.9.3",
    "react-router": "4.0.0-alpha.4",
    "react-router-dom": "^4.1.1",
    "react-select": "^1.2.1",
    "react-slick": "^0.15.4",
    "react-waypoint": "^7.3.2",
    "reactstrap": "5.0.0-beta",
    "redux": "^3.7.2",
    "redux-logger": "^3.0.6",
    "redux-promise-middleware": "^4.4.2",
    "redux-thunk": "^2.2.0",
    "uuid": "^3.2.1"
  }```

**Comment by lukechilds** [comment]
Are you sure this error is related to `react-jsdom`?

**Comment by rei-gun** [comment]
The error only occurs when I import react-jsdom

**Comment by lukechilds** [comment]
Can you paste the full error?

**Comment by tomcardoso** [comment]
Getting this same error. In my case, I'm using it in a Meteor project. Here's the full error message:
```
W20180406-14:54:14.032(-4)? (STDERR) /Users/tcardoso/.meteor/packages/meteor-tool/.1.6.1.1shkhlm++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20180406-14:54:14.032(-4)? (STDERR) 						throw(ex);
W20180406-14:54:14.033(-4)? (STDERR) 						^
W20180406-14:54:14.033(-4)? (STDERR)
W20180406-14:54:14.033(-4)? (STDERR) Error: Cannot find module 'react/lib/ReactComponentTreeHook'
W20180406-14:54:14.033(-4)? (STDERR)     at Function.Module._resolveFilename (module.js:538:15)
W20180406-14:54:14.034(-4)? (STDERR)     at Function.Module._load (module.js:468:25)
W20180406-14:54:14.034(-4)? (STDERR)     at Module.require (module.js:587:17)
W20180406-14:54:14.034(-4)? (STDERR)     at require (internal/module.js:11:18)
W20180406-14:54:14.034(-4)? (STDERR)     at Object.<anonymous> (/Users/tcardoso/Sites/globe/chart-tool/meteor/node_modules/react-jsdom/node_modules/react-dom/lib/ReactDebugTool.js:14:30)
W20180406-14:54:14.034(-4)? (STDERR)     at Module._compile (module.js:643:30)
W20180406-14:54:14.035(-4)? (STDERR)     at Object.Module._extensions..js (module.js:654:10)
W20180406-14:54:14.035(-4)? (STDERR)     at Module.load (module.js:556:32)
W20180406-14:54:14.035(-4)? (STDERR)     at tryModuleLoad (module.js:499:12)
W20180406-14:54:14.035(-4)? (STDERR)     at Function.Module._load (module.js:491:3)
W20180406-14:54:14.036(-4)? (STDERR)     at Module.require (module.js:587:17)
W20180406-14:54:14.036(-4)? (STDERR)     at require (internal/module.js:11:18)
W20180406-14:54:14.036(-4)? (STDERR)     at Object.<anonymous> (/Users/tcardoso/Sites/globe/chart-tool/meteor/node_modules/react-jsdom/node_modules/react-dom/lib/ReactInstrumentation.js:17:24)
W20180406-14:54:14.036(-4)? (STDERR)     at Module._compile (module.js:643:30)
W20180406-14:54:14.037(-4)? (STDERR)     at Object.Module._extensions..js (module.js:654:10)
W20180406-14:54:14.037(-4)? (STDERR)     at Module.load (module.js:556:32)
W20180406-14:54:14.037(-4)? (STDERR)     at tryModuleLoad (module.js:499:12)
W20180406-14:54:14.038(-4)? (STDERR)     at Function.Module._load (module.js:491:3)
W20180406-14:54:14.038(-4)? (STDERR)     at Module.require (module.js:587:17)
W20180406-14:54:14.038(-4)? (STDERR)     at require (internal/module.js:11:18)
W20180406-14:54:14.038(-4)? (STDERR)     at Object.<anonymous> (/Users/tcardoso/Sites/globe/chart-tool/meteor/node_modules/react-jsdom/node_modules/react-dom/lib/ReactReconciler.js:12:28)
W20180406-14:54:14.039(-4)? (STDERR)     at Module._compile (module.js:643:30)
```

**Comment by tomcardoso** [comment]
I just forked this repo and updated the React dependencies to React 16, all works now, FYI

**Comment by lukechilds** [comment]
Thanks @tomcardoso, are you able to submit a PR?

**Comment by lukechilds** [comment]
This should be resolved now in `react-jsdom@3.0.0`, apologies for the delay.
